src/scripts/blocks/post-list/DisplayComponent.js

```diff
@@ -104,7 +108,22 @@ class DisplayComponent extends Component {
               label={ __('Use related categories where supported', 'benenson') }
               checked={ attributes.categoryRelated }
               onChange={ this.createUpdateAttribute('categoryRelated') }
-            />}
+            /> }
+            { attributes.type === 'select' && <ToggleControl
+              label={ __('Display excerpt', 'benenson') }
+              checked={ attributes.displayExcerpt }
+              onChange={ this.createUpdateAttribute('displayExcerpt') }
+            /> }
+            { attributes.type === 'select' && (<div>
+              <label name="editorButtonText">{ __('Button Text:', 'benenson') } </label>
```

Comment:
Revert this change; https://github.com/bigbitecreative/benenson/pull/53#discussion_r245980947 refers to L120, not L118.